### ISSUE TITLE
Module parse failed: Identifier '_templateObject' has already been declared

### ISSUE THREAD

**Issue by Lee-Jason-Curley** [wallabyjs/console-ninja#249]
### Issue description or question

I can't run Console Ninja without running into this issue outlined in Stackoverflow post:
[https://stackoverflow.com/questions/74834358/module-parse-failed-identifier-templateobject-has-already-been-declared](https://stackoverflow.com/questions/74834358/module-parse-failed-identifier-templateobject-has-already-been-declared)

Please note that this has been viewed by 1000+ times. So, it seems to be a persistent issue. My app is using node 18.14.0.

Is there going to be fix anytime soon? As is, Console Ninja is unusable for me. I'm using v1.0.268.

**Comment by xuxucode** [comment]
See sample repo: https://github.com/xuxucode/console-ninja-issue-249/blob/master/app/page.tsx#L12-L14

The critical part is a class with private methods:
```ts
// A class with two private methods which have same names. The only difference
// is one of them begins with a underscore.
class Client {
  #privateMethod() { }

  #_privateMethod() { }
}
```

**Comment by smcenlly** [comment]
@xuxucode - We can reproduce the same issue in your sample repo with Console Ninja paused, disabled, and when Console Ninja is not installed. The problem in your sample repo is not a problem with Console Ninja, it looks to be a problem with Next.js compilation.

**Comment by xuxucode** [comment]
@smcenlly Yes, it is related to Next.js, has nothing to do with Console Ninja.

**Comment by xuxucode** [comment]
I also post the comment at https://github.com/vercel/next.js/issues/44113#issuecomment-1898503310, maybe it should be fixed there.
